src/IntelliTect.Coalesce.Tests/Tests/Utilities/StringExtensionTests.cs

```diff
@@ -84,5 +85,37 @@ public void CanGetIndetifierFromVerbatimIdentifier()
 
             Assert.Equal("@if", identifier);
         }
+
+        [Fact]
+        public void ToProperCase_HandlesTwoCapitalsInARow()
+        {
+            const string enumName = "HROnly";
+            Assert.Equal("HR Only", enumName.ToProperCase());
+        }
+        [Fact]
+        public void ToProperCase_HandlesMultipleCapitalsInARow()
+        {
+            const string enumName = "FBIOnly";
+            Assert.Equal("FBI Only", enumName.ToProperCase());
+        }
+        [Fact]
+        public void ToProperCase_HandlesOnlyTwoCapitals()
+        {
+            const string enumName = "UI";
+            Assert.Equal("UI", enumName.ToProperCase());
+        }
+        [Fact]
+        public void ToProperCase_HandlesANormalWord()
+        {
+            const string enumName = "User";
+            Assert.Equal("User", enumName.ToProperCase());
+        }
+
```

Comment:
Can you add some tests for camelCase inputs too? They use this method too, e.g. for method parameters.

Reply:
Done!